### ISSUE TITLE
[BUG] Unable to pair 9.00 PS4 on windows and steam deck

### ISSUE THREAD

**Issue by GingerBobRoss** [streetpea/chiaki-ng#256]
**Describe the bug**
 Unable to pair 9.00 PS4 on windows and steam deck. PS4 generates a CE-35407-9 error and Chiaki AND Chiaki4Deck gives off 0x80180b03 error code

**To Reproduce**
insert PSN ID/ Base64 ID
Insert Pin from PS4
Click register 

**Expected behavior**
System should register Console and launch Remote Access 

**Desktop (please complete the following information):**
 - OS: [Steam OS 3.5 | Windows 10]
  - Device [Steam Deck & PC ]
 - Chiaki version 2.2.0 and Chiaki4Deck 1.6.3

**Comment by jbaiter** [comment]
Please provide a debug log.

**Comment by GingerBobRoss** [comment]
Ive tried enabling Verbose logging and its not creating logs in the logs folder.
Ill try to completely uninstall Chiaki again and remove any left overs and try again.

**Comment by streetpea** [comment]
Logs are only saved for streaming sessions. Since this is before a stream session please launch Chiaki4deck via the konsole using flatpak run io.github.streetpea.Chiaki4deck and record the log output in the konsole

**Comment by GingerBobRoss** [comment]
> Logs are only saved for streaming sessions. Since this is before a stream session please launch Chiaki4deck via the konsole using flatpak run io.github.streetpea.Chiaki4deck and record the log output in the konsole

Here is the log; it reports a network socket error
[Chiaki log.txt](https://github.com/streetpea/chiaki4deck/files/14394245/Chiaki.log.txt)

**Comment by streetpea** [comment]
Seems you are getting unauthorized ... Are you entering the correct remote play code from the console and have remote play enabled in the console settings?

**Comment by streetpea** [comment]
Also make sure the account id you're using is for the user that is currently logged into the system when you request the remote play code

**Comment by GingerBobRoss** [comment]
yeah, im doing everything ive done before in the past and havent had any issues
The pin doesnt allow spaces so that shouldnt be the issue either. I really dont know whats going on

**Comment by streetpea** [comment]
@GingerBobRoss @XargonWan have you tried with previous versions of Chiaki-ng such as 1.6.3?
